node/init.go

```diff
@@ -0,0 +1,130 @@
+package node
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/celestiaorg/celestia-node/core"
+	"github.com/celestiaorg/celestia-node/libs/fslock"
+	"github.com/celestiaorg/celestia-node/libs/utils"
+)
+
+// Init initializes the Node FileSystem(FS) Repository for th given Node Type 'tp' in the directory under 'path' with
+// the given Config.
+func Init(path string, tp Type) error {
```

Comment:
Can we open an issue about re-using the (base)apps init logic here later?

Reply:
Ok